### ISSUE TITLE
No module named 'Bio'

### ISSUE THREAD

**Issue by yang3kc** [menchelab/CartoGRAPHs#2]
I got the following error message when I tried to import the package:

```py
ModuleNotFoundError: No module named 'Bio'
```

which was thrown by 
```py
unc_calculations.py in <module>
     10 import networkx as nx
     11 
---> 12 from Bio import Entrez
     13 import pymysql as mysql
     14 import pandas as pd
```

Not sure what lib this is and the setup file doesn't specify it either.

**Comment by chris-huetter** [comment]
Thank you for trying out the package and pointing us towards the issue! We have just updated a few minor changes to pypi (now available for installing is v1.0.1) and the github repo, which is now also listing the biopython package in the requirements file.

**Comment by yang3kc** [comment]
Thank you!
